scripts/steamed_hams.py

```diff
@@ -1,21 +1,21 @@
 #!/usr/bin/python3
 
 """
-Copyright SuperDARN Canada 2020
-Keith Kotyk
+    Borealis Start up script
+    ~~~~~~~~~~~~~~~~~~~~~~~~
 
-Borealis Start up script
-
-The Simpsons, Season 1 Episode 3 - Steamed Hams
+    The Simpsons, Season 7 Episode 21
 
+    :copyright: 2020 SuperDARN Canada
+    :author: Keith Kotyk
 """
 import argparse
 import sys
 import subprocess as sp
 import datetime
 import os
 import time
-from src.utils.general import load_config
+from general import load_config
```

Comment:
If this is not the only place outside src where we need load_config() maybe we can just stick it in here and remove general.py all together. We don't have any other script utils do we?